ctru-rs/src/services/gfx.rs

```diff
@@ -138,7 +166,7 @@ impl<S: Screen> Flush for S {
         let framebuffer = self.raw_framebuffer();
 
         // Flush the data array. `self.raw_framebuffer` should get the correct parameters for all kinds of screens
-        unsafe {
+        let _ = unsafe {
```

Comment:
huh. Should this function be returning a `Result`? I wonder what scenarios flushing could fail like this...

Reply:
I guess it could fail if the pointers are invalid, but given the guarantees made by the lifetime system, it shouldn't be a problem.

Reply:
Ah yeah that makes sense, The error here looks like it would normally only fail if the address is not in linear memory (which for these frame buffers should never happen as far as I know): https://www.3dbrew.org/wiki/GSPGPU:FlushDataCache